src/dom/README.md

```diff
@@ -264,3 +264,140 @@ assert.isTrue(string === "" + ionString);
 assert.isTrue(string === ionString.stringValue());
 assert.isFalse(ionString === ionString2);
 ```
+
+### `dom.Value` equivalence
+
+dom.Value object equivalence is implemented by two methods ionEquals(compares two dom.Value objects based on strict or non-strict comparison type) and equals(compares a dom.Value object with a corresponding JS object).
+
+#### `ionEquals`:
+
+This method compares a dom.Value object with another dom.Value object with either strict or non-strict comparison type. 
+
+*Strict equivalence* refers to Ion data model equivalence as defined in Ion Equivalence (https://www.javadoc.io/doc/com.amazon.ion/ion-java/latest/com/amazon/ion/util/Equivalence.html) and by Ion Specification (http://amzn.github.io/ion-docs/docs/spec.html)
+*Structural or non-strict equivalence* follows the same rules as strict equivalence,except that
+
+* 1. Annotations are not considered, and
+* 2. Timestamps that represent the same instant in time are always considered equivalent.
+
+#### `equals`:
+
+This method compares a dom.Value object with a corresponding JS object. 
+
+#### Integer Example:
+```javascript
+let int1: Value = load("foo::bar::7");
+let int2: Value = load("foo::bar::7");
+let int3: Value = load("7");
+
+// Equivalence between two Ion DOM Values
+int1.ionEquals(int2); // returns true
+int1.ionEquals(int3); // returns false as annotations didn't match in a strict equivalence
+int1.ionEquals(int3, {ignoreAnnotations: true}); // returns true for non-strict equivalence
+
+// Equivalence between JS Value and Ion DOM Value
+int1.equals(7); // returns true
+int1.ionEquals(7); // returns false
+```
+
+#### Float Example:
+```javascript
+let float1: Value = load("baz::qux::15e-1");
+let float2: Value = load("baz::qux::15e-1");
+let float3: Value = load("15e-1");
+let float4: Value = load("1.5");
+let float5: Value = load("12e-1");
+
+// Equivalence between two Ion DOM Values
+float1.ionEquals(float2); // returns true
+float1.ionEquals(float3); // return false as annotations didn't match in a strict equivalence
+
+float1.ionEquals(float4); // returns false as Decimal and Float values will not be equivalent
+float3.ionEquals(float5, {epsilon: 0.5}); // returns true as both values are equivalent by epsilon value
+float3.ionEquals(float5, {epsilon: 0.2}); // returns false as both values are not equivalent by epsilon value
+
+// Equivalence between JS Value and Ion DOM Value
+float1.equals(1.5); // returns true
+float1.ionEquals(1.5); // returns false
```

Comment:
I think this particularly this example (and also the preceding one) would be clearer without variables. Reading this I'm bouncing back and forth constantly from definitions for `floatN` and their uses.

If we must have variables names, let the names be descriptive and helpful. `same1`, `same2`, `unannotated`, `ion_float`, `ion_decimal`, `nearby_float`... something like that?

Reply:
Yes, I think we can have variable names descriptive for making it more readable. I will add this change along with other typos and minor changes.